bazarr/sonarr/sync/parser.py

```diff
@@ -12,7 +12,17 @@
 from .converter import SonarrFormatVideoCodec, SonarrFormatAudioCodec
 
 
-def seriesParser(show, action, tags_dict, serie_default_profile, audio_profiles):
+def get_matching_profile(tags, language_profiles):
+    matching_profile = None
+    if len(tags) > 0:
+        for _, tuple in enumerate(language_profiles):
+            if tuple[2] in tags:
+                matching_profile = tuple[0]
+                break
```

Comment:
Just a thought maybe this is easier to read:

```suggestion
   if len(tags) == 0:
     return None

    for _, tuple in enumerate(language_profiles):
        if tuple[2] in tags:
            return tuple[0]
```

Reply:
It is arguable as to which version is easier to read.
However, I feel the original version is better for use with an interactive debugger or debugging in general.
If I want to examine the return value within the debugger, then I only need to place a single breakpoint.
Or if I wanted simply to print the value, I would only need a single print statement.
In general, I prefer my methods to have single exit points, where feasible, for the reasons above. Also, having the current potential return value always stored in a variable is a benefit in larger and more complex methods, as you can place a watch on that variable to see how it changes through the lifetime of the method.
I realize the method above is a small one, but small methods often grow to become larger methods. 😄